controllers/grid/users/reviewer/form/ReviewerForm.php

```diff
@@ -73,6 +73,17 @@ public function __construct($submission, $reviewRound)
         $this->addCheck(new \PKP\form\validation\FormValidator($this, 'responseDueDate', 'required', 'editor.review.errorAddingReviewer'));
         $this->addCheck(new \PKP\form\validation\FormValidator($this, 'reviewDueDate', 'required', 'editor.review.errorAddingReviewer'));
 
+        $this->addCheck(
+            new \PKP\form\validation\FormValidatorDateCompare(
+                $this,
+                'reviewDueDate',
+                \Carbon\Carbon::parse(Application::get()->getRequest()->getUserVar('responseDueDate')),
```

Comment:
May we use `$this->getData('responseDueDate')` here?

Reply:
fixed.